File: testdata/dirty/sample.go
package testdata

import "fmt"

// BARK: This is a test function that needs cleanup
func TestFunction() {
	fmt.Println("Hello, World!")
	// BARK: Remove this debug code before commit
	fmt.Println("Debug information")
	// BARK plain marker without colon
	x := 42
}

// Regular comment without BARK
func AnotherFunction() {
	// BARK: TODO fix this hack
	x := 42
	// BARK
	fmt.Println(x)
}

File: testdata/dirty/sample.c
#include <stdio.h>

// BARK: Remove debug printf statements
// BARK
int main() {
    printf("Debug mode\n");  // BARK: Delete this line
    
    /* Regular comment */
    int x = 42;
    
    /* BARK: This malloc needs proper error handling */
    int* ptr = malloc(sizeof(int) * 100);
    
    // BARK plain test
    free(ptr);

    return 0;
}

